### ISSUE TITLE
no CA certs in image

### ISSUE THREAD

**Issue by cmosetick** [cmattoon/aws-ssm#7]
Seems like we need volume mount CA certs or provide some in the docker image:

```level=info msg="Couldn't get value for aws-ssm/test-secret: RequestError: send request failed\ncaused by: Post https://ssm.us-east-2.amazonaws.com/: x509: failed to load system roots and no roots provided"
```

**Comment by cmosetick** [comment]
PR #8 adds ca certificates to the aws-ssm docker image.

I'm aware that some people think its bad practice to include CA certs in docker images, but it seems to make sense in this situation, rather than using host volume mounting of /etc/ssl/certs.

**Comment by cmattoon** [comment]
Agreed on the volume mounts, though maybe I can add that to the chart later if needed. Merged #8, so we should be good now. Will do some more testing and tag another release later. Thanks!
